push_feedback_to_all.sh

```diff
@@ -5,14 +5,18 @@
 feedback_filename=$1
 
 for dir in */; do
-    if [[ -d $dir ]]; then
-    	echo "Pushing for $dir"
-       	cd $dir
-        git checkout master
-        git add $1
-        git commit $1 -m "Grade assignment"
-        git pull --rebase
-        git push
-        cd ..
-    fi
+    cd $dir
+    for subdir in */; do
+        if [[ -d $subdir ]]; then
```

Comment:
You changed something about the expected directory structure.. as per my previous comment, can you make it clear what structure the script expects